src/substrait/__init__.pyi

```diff
@@ -0,0 +1 @@
+from . import proto
```

Comment:
I was trying to see if we could move the version info to this `.pyi` file you have, but it seems like pyi files are only supposed to have type information. does it make sense to have this stub file if there's only this import?

Reply:
I don't think so. It gets autogenerated from `protol` I believe. It used to generate .pyi files for all the classes, but stopped (maybe my version was updated locally mid-development?). Maybe we just add `*.pyi` to .gitignore.